conan/tools/gnu/makedeps.py

```diff
@@ -507,8 +512,9 @@ class DepGenerator:
     Process a dependency cpp_info variables and generate its Makefile content
     """
 
-    def __init__(self, dependency):
+    def __init__(self, dependency, require=None):
```

Comment:
If the ``require`` is always passed, better remove the ``None`` default

Reply:
Whoops, minor oversight there. Was thinking of making it optional at first.